modules/gapi/src/backends/oak/oak_memory_adapters.hpp

```diff
@@ -28,6 +29,21 @@ class GAPI_EXPORTS OAKMediaAdapter final : public cv::MediaFrame::IAdapter {
     std::vector<uint8_t> m_buffer;
 };
 
+class GAPI_EXPORTS OAKRMatAdapter final : public cv::RMat::Adapter {
```

Comment:
Please add a brief description here on this class purpose

Reply:
Done